tests/components/sensoterra/test_config_flow.py

```diff
@@ -0,0 +1,148 @@
+"""Test the Sensoterra config flow."""
+
+from unittest.mock import AsyncMock, patch
+
+from sensoterra.customerapi import InvalidAuth as StInvalidAuth, Timeout as StTimeout
+
+from homeassistant import config_entries
+from homeassistant.components.sensoterra.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, CONF_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+API_ACCESS_TOKEN = "api_token"
+API_EMAIL = "test-email@example.com"
+API_PASSWORD = "test-password"
+HASS_UUID = "phony-unique-id"
+
+
+async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] == {}
+
+    hass.data["core.uuid"] = HASS_UUID
+
+    with patch(
+        "sensoterra.customerapi.CustomerApi.get_token",
+        return_value=API_ACCESS_TOKEN,
+    ):
```

Comment:
Let's create a global mock so we can avoid having to patch this inline every time. Checkout `airgradient` for an example